src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    private final String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean isDone() {
+        return this.isDone;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + (this.isDone ? "X" : " ") + "] " + this.description;
+    }
+}
```

Comment:
Maybe empty line at the end of files, so it would be known when to terminate the read